src/test/java/com/atlassian/jira/cloud/jenkins/common/client/JenkinsAppApiTest.java

```diff
@@ -34,7 +38,10 @@
 
     /** Generates an example JWT that can be used in other tests. */
     @Test
-    public void generateExampleJwt() throws IOException {
+    public void generateExampleBuildsJwt() throws IOException {
```

Comment:
Some `generateExampleJwt` methods so we can generate JWTs for testing purposes. I used these for the tests in the Forge app, for example.